tests/copier_data/data1.yaml

```
@@ -7,6 +7,8 @@ install_claude_cli: false
 ssh_port_number: 12345
 
 
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (4 > 2)

(empty-lines)